seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/schema/TableSchemaOptions.java

```diff
@@ -49,6 +49,7 @@ public static class TableIdentifierOptions {
                         .withDescription("SeaTunnel Schema Table Comment");
     }
 
+    @Deprecated
```

Comment:
This option not only for ElasticSearch, so we can not mark it as deprecated. I think we should print warning log when user use schema config in elasticsearch source in https://github.com/apache/seatunnel/blob/24441c876d28fb26d6e820628197e3682092363c/seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/source/ElasticsearchSource.java#L61
```suggestion
```